t/mime.rakutest

```diff
@@ -7,13 +7,16 @@ use lib 'lib';
 
 use MIME::Types;
 
-my $mime = MIME::Types.new('./doc/mime.types');
+my $mime = MIME::Types.new('./resources/mime.types');
```

Comment:
Resources need to be dealt with specifically, using the %?RESOURCES variable; that's its whole point, actually...

Reply:
Ahh I see, in my test `my $mime = MIME::Types.new(%?RESOURCES<mime.types>);`  fails, seems that `%?RESOURCES<mime.types>` is `Nil` would you expect it to be populated when used inside a test?

Reply:
The thing is that we really don't know where the lib actually is when we run the tests. It could be already installed, so it could have a non-Nil value. So it's probably better to have a check on the value of the resource, if available, and then use the files in the same filesystem; just to be on the safe side, check if the file is in ../resources too. (you might want to run, for debugging purposes or whatever, the test from the same directory)